### PR TITLE
Add Release note for KubeDB v2024.8.21

### PR DESCRIPTION
**Pull request by souravbiswassanto**
Signed-off-by: souravbiswassanto <saurov@appscode.com>

### CODE REVIEW

File: content/post/kubedb-v2024.08.16/index.md [hunk 1]
```diff
@@ -0,0 +1,1689 @@
...
+          requests:
+            storage: 10Gi
+        storageClassName: standard
+  storageType: Durable
+  deletionPolicy: WipeOut
+```
+
+
+
+## Kafka Schema Registry
```
Comment (obaydullahmhs) [inline content/post/kubedb-v2024.08.16/index.md:280]: Kafka Rest Proxy

Comment (souravbiswassanto) [reply content/post/kubedb-v2024.08.16/index.md]: updated

File: content/post/kubedb-v2024.08.16/index.md [hunk 2]
```diff
@@ -0,0 +1,1689 @@
+---
+title: Announcing KubeDB v2024.08.16
+date: ""
```
Comment (mohin7) [inline content/post/kubedb-v2024.08.16/index.md:3]: Add valid date
Format "2024-08-19"

Comment (souravbiswassanto) [reply content/post/kubedb-v2024.08.16/index.md:3]: Vai this will be adjusted once release date is fixed.

**Review by pritamdas99 — APPROVED** [review]
LGTM!

File: content/post/kubedb-v2024.08.16/index.md [hunk 3]
```diff
@@ -0,0 +1,1689 @@
...
+  topology:
+    routers:
+      replicas: 1
+```
+#### Deploying Druid with TLS secure MySQL and PostgreSQL
+Additionally, in this release, KubeDB  has been enhanced to support integration of `Druid` with TLS-secured `MySQL` and `PostgreSQL` clusters.
+
+## Elasticsearch
+
+In this release, In this release, API version for Elasticsearch has been upgraded to `v1`. The new API version is [kubedb.com/v1](http://kubedb.com/v1).
```
Comment (Neaj-Morshad-101) [inline content/post/kubedb-v2024.08.16/index.md:163]: remove duplicate. `In this release, In this release`

File: content/post/kubedb-v2024.08.16/index.md [hunk 4]
```diff
@@ -0,0 +1,1689 @@
...
+
+1. In `v1alpha2` API, resources for dedicated topology cluster resources have to be defined at `.spec.topology.<node-type>.resources`, NodeSelector labels have to be defined at `.spec.topology.<node-type>.nodeSelector` and Tolerations have to be defined at `.spec.topology.<node-type>.tolerations.
+Now, in `v1` API version resources has to be defined in `.spec.topology.<node-type>.podTemplate.containers[].resources`. NodeSelector labels have to be defined at `.spec.topology.<node-type>.podTemplate.containers[].nodeSelector` and Tolerations have to be defined at `..spec.topology.<node-type>.podTemplate.containers[].tolerations.
```
Comment (Neaj-Morshad-101) [inline content/post/kubedb-v2024.08.16/index.md:166]: `.spec.topology.<node-type>.podTemplate.containers[].nodeSelector` -> `.spec.topology.<node-type>.podTemplate.spec.nodeSelector`

`..spec.topology.<node-type>.podTemplate.containers[].tolerations. `->  `.spec.topology.<node-type>.podTemplate.spec.tolerations`.

**Review by Neaj-Morshad-101 — CHANGES_REQUESTED** [review]

**Comment by souravbiswassanto** [comment]
```
druid 30.0.0

-------------------------------
ferretdb 1.23.0


-------------------------------
singlestore

alma-8.7.10-95e2357384
alma-8.5.30-4f46ab16a5

-------------------------------
Pgpool 4.5.3
Pgpool 4.4.8

-------------------------------
mariadb: (cherry-pick)

11.3.2-jammy
11.4.3-noble
11.5.2-noble

memcached: (cherry-pick)

1.6.29-alpine

redis:

7.4.0-alpine
7.4.0-bookworm


mysql: (cherry-pick)

8.4.2-oracle
9.0.1-oracle


Postgres:

16.4-alpine
16.4-bookworm
15.8-alpine
15.8-bookworm
14.13-alpine
14.13-bookworm
```
This need to be added.

File: content/post/kubedb-v2024.08.16/index.md [hunk 5]
```diff
@@ -0,0 +1,1826 @@
...
+- schema-registry
+- security
+- singlestore
+- solr
+- tls
+- zookeeper
+---
+
+We are excited to announce the release of KubeDB v2024.7.x! This release introduces support for the kubedb.com/v1 APIVersion across the following KubeDB-supported databases:
+- PostgreSQL
```
Comment (Neaj-Morshad-101) [inline content/post/kubedb-v2024.08.16/index.md:43]: Add ProxySQL

**Review by Neaj-Morshad-101 — CHANGES_REQUESTED** [review]

File: content/post/kubedb-v2024.08.16/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (souravbiswassanto) [reply content/post/kubedb-v2024.08.16/index.md:163]: Removed

File: content/post/kubedb-v2024.08.16/index.md (hunk 4 above)
Comment (souravbiswassanto) [reply content/post/kubedb-v2024.08.16/index.md:166]: what does that mean? @Neaj-Morshad-101

**Comment by souravbiswassanto** [comment]
> ```
> druid 30.0.0
> 
> -------------------------------
> ferretdb 1.23.0
> 
> 
> -------------------------------
> singlestore
> 
> alma-8.7.10-95e2357384
> alma-8.5.30-4f46ab16a5
> 
> -------------------------------
> Pgpool 4.5.3
> Pgpool 4.4.8
> 
> -------------------------------
> mariadb: (cherry-pick)
> 
> 11.3.2-jammy
> 11.4.3-noble
> 11.5.2-noble
> 
> memcached: (cherry-pick)
> 
> 1.6.29-alpine
> 
> redis:
> 
> 7.4.0-alpine
> 7.4.0-bookworm
> 
> 
> mysql: (cherry-pick)
> 
> 8.4.2-oracle
> 9.0.1-oracle
> 
> 
> Postgres:
> 
> 16.4-alpine
> 16.4-bookworm
> 15.8-alpine
> 15.8-bookworm
> 14.13-alpine
> 14.13-bookworm
> ```
> 
> This need to be added.

Done adding

File: content/post/kubedb-v2024.08.16/index.md (hunk 4 above)
Comment (souravbiswassanto) [reply content/post/kubedb-v2024.08.16/index.md:178]: @raihankhan vai can you review this and update if needed?

**Review by MobarakHsn — APPROVED** [review]
LGTM!

**Review by sheikh-arman — COMMENTED** [review]
lgtm

**Review by Neaj-Morshad-101 — DISMISSED** [review]

**Review by sayedppqq — COMMENTED** [review]
LGTM

**Review by MobarakHsn — APPROVED** [review]

**Comment by HiranmoyChowdhury** [comment]
LGTM

**Review by HiranmoyChowdhury — DISMISSED** [review]

File: content/post/kubedb-v2024.8.21/index.md [hunk 6]
```diff
@@ -0,0 +1,1859 @@
...
+To deploy a Druid cluster that includes an internally managed MySQL database and ZooKeeper instance, use the following YAML configuration:
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Druid
+metadata:
+  name: druid-quickstart
+  namespace: demo
+spec:
+  version: 28.0.1
```
Comment (tapojit047) [inline content/post/kubedb-v2024.8.21/index.md:169]: 30.0.0

File: content/post/kubedb-v2024.8.21/index.md [hunk 7]
```diff
@@ -0,0 +1,1859 @@
...
+
+If you prefer to use PostgreSQL for metadata storage instead of the default MySQL, simply specify it in the `metadataStorage.type` field as shown below:
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Druid
+metadata:
+  name: druid-quickstart
+  namespace: demo
+spec:
+  version: 28.0.1
```
Comment (tapojit047) [inline content/post/kubedb-v2024.8.21/index.md:187]: 30.0.0

Comment (souravbiswassanto) [reply content/post/kubedb-v2024.8.21/index.md:187]: change and push

**Review by Evanraisul — APPROVED** [review]
LGTM

**Review by raihankhan — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**